plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ServletClasspathProvider.java

```diff
@@ -30,7 +30,7 @@ public ServletClasspathProvider() {
 
   @Override
   public IClasspathEntry[] resolveClasspathContainer(IRuntime runtime) {
-    java.nio.file.Path cloudSdkPath = new CloudSdkProvider().getCloudSdkLocation().toPath();
+    java.nio.file.Path cloudSdkPath = new CloudSdk.Builder().build().getSdkPath();
```

Comment:
This will be fixed in https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/335, after I resolve the Travis issues there.